autoload/neomake/makers/ft/erlang.vim

```diff
@@ -10,3 +10,39 @@ function! neomake#makers#ft#erlang#erlc() abort
             \ '%E%f:%l: %m'
         \ }
 endfunction
+
+function! neomake#makers#ft#erlang#erlc_glob_paths() abort
+    return {
+        \ 'exe': 'erlc',
+        \ 'args': function('neomake#makers#ft#erlang#glob_paths'),
+        \ 'errorformat':
+            \ '%W%f:%l: Warning: %m,' .
+            \ '%E%f:%l: %m'
+        \ }
+endfunction
+
+function! neomake#makers#ft#erlang#glob_paths() abort
+    if match(expand('%'), 'SUITE.erl$') > -1
```

Comment:
Check out `fnamemodify()`, i.e. `fnamemodify('%:t)` to get the basename, if that's what you want.

Reply:
Common Test, a testing framework, defines a naming convention for files  containing tests - they should end with `_SUITE.erl`. In this particular case it doesn't matter if we test on the basename or the whole path.

Reply:
I see.  Thought like the file itself needs to be `SUITE.erl`.
You may want to include the `_` then maybe?

Also `if expand('%') =~# '_SUITE.erl$'` could be used.